src/common.h

```diff
@@ -31,6 +31,34 @@ extern "C" {
 	"Please do not compile this with a lslboost version older than 1.45 because the library would otherwise not be protocol-compatible with builds using other versions."
 #endif
 
+// size of a cache line
```

Comment:
This block should be in `consumer_queue.h`

Reply:
Sounds good.

Reply:
Ah well there's one thing -- we'll need that definition in `sample.h` too for the same reason (the 2 atomics in the factory there are prone to false sharing).